### PR TITLE
Add round of service docs

### PR DESCRIPTION
**Pull request by ivalaginja**
Adding some more docs to some of the services as listed in #190.

Also adds them all to the list in `index.rst`.

### CODE REVIEW

**Comment by ivalaginja** [comment]
@raphaelpclt this is ready for review

File: docs/services/aimtti_plp.rst [hunk 1]
```diff
@@ -1,20 +1,72 @@
...
+
+- The device automatically applies a remote interface lock when it is commanded for the first time
+(see `manual <https://resources.aimtti.com/manuals/New_PL+PL-P_Series_Instruction_Manual-Iss18.pdf>`_, page 23).
+This is also noted in the `dcps library <https://github.com/sgoadhouse/dcps/blob/afbe687236bfa6176240e26790dd26b6c395b515/dcps/AimTTiPLP.py#L85>`_.
+Even after setting the interface lock to ``LOCAL``, this will instantly be overwritten back to ``REMOTE`` when a new command is issued.
+- All remote commands are listed on page 34 of the `device manual <https://resources.aimtti.com/manuals/New_PL+PL-P_Series_Instruction_Manual-Iss18.pdf>`_.
+- Only a minimum of the commands listed in the manual are implemented. More commands can easily be added in the future as needed.
+
+The service has been successfully tested with the following device:
+
```
Comment (raphaelpclt) [inline docs/services/aimtti_plp.rst:24]: Shall we precise the platform on which it was tested?

**Review by raphaelpclt — APPROVED** [review]
Looks good

File: docs/services/aimtti_plp.rst (hunk 1 above)
Comment (ivalaginja) [reply docs/services/aimtti_plp.rst:24]: Hmm this is a good point. We know that everything was only run on Windows for now though, so let's ponder on this independently.

**Pull request merged**